prow/cmd/tide/main.go

```diff
@@ -50,6 +55,13 @@ type options struct {
 	runOnce    bool
 	kubernetes prowflagutil.KubernetesOptions
 	github     prowflagutil.GitHubOptions
+
+	maxRecordsPerPool int
+
+	// The following are used for persisting action history in GCS.
```

Comment:
Do we want to add some doc to be prescriptive about where to put this? Should people have a separate bucket from test results? Will this leak data for hidden pools? Is it safe to put this into a public bucket?

Reply:
Good point. I updated this comment and the flag help text to indicate that the location should not be publicly readable if any repos are sensitive, but otherwise can be anything.